php/src/AbstractJws.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace TrueLayer\Signing;
+
+use TrueLayer\Signing\Contracts\Jws as IJws;
+use TrueLayer\Signing\Exceptions\RequestPathNotFoundException;
+
+abstract class AbstractJws implements IJws
+{
+    protected string $request_method = 'POST';
```

Comment:
Looks like there is a mixture of snake case and camel case in the code? Would recommend sticking to one, camel case would be my choice.